simplivity/resources/omnistack_clusters.py

```diff
@@ -110,3 +110,9 @@ def __init__(self, connection, resource_client, data):
         self.data = data
         self._connection = connection
         self._client = resource_client
+
+    def get_time_zone_list(self):
```

Comment:
This method should be part of the OmniStackClusters class as this operation is not for a specific Omnistack cluster object.

Reply:
Yes true code must have need OmniStackClusters class, Updated the code as per suggestion.